test cases/frameworks/23 hotdoc/meson.build

```diff
@@ -0,0 +1,9 @@
+project('hotdoc', 'c')
```

Comment:
Should probably specify a version here, then use meson.project_version() in generate_doc()

Reply:
Would we want that to be automatic if not `project_version` is not specified?

Reply:
That sounds like a good idea, but it might lead to confusing bugs in practice, as most projects would want to use the major version here

Reply:
Done